code/modules/mining/lavaland/necropolis_chests.dm

```diff
@@ -1128,7 +1128,7 @@
 	var/blast_range = 13 //how long the cardinal blast's walls are
 	var/obj/effect/hierophant/beacon //the associated beacon we teleport to
 	var/teleporting = FALSE //if we ARE teleporting
-	var/friendly_fire_check = FALSE //if the blasts we make will consider our faction against the faction of hit targets
+	var/friendly_fire_check = TRUE //if the blasts we make will consider our faction against the faction of hit targets
```

Comment:
Why even keep this in?

Reply:
Admins can still override it if they choose